src/main/java/pl/arturzgodka/apihandlers/BaseUrlParts.java

```diff
@@ -3,11 +3,17 @@
 public class BaseUrlParts {
```

Comment:
why static getters for static fields?

Reply:
Otherwise errors occures, while I want to access non-static fields from static getters.

Reply:
ok, they can be just "public static" fields.

Reply:
done